src/xenia/app/profile_dialogs.h

```diff
@@ -54,19 +55,18 @@ class ProfileConfigDialog final : public ui::ImGuiDialog {
  public:
   ProfileConfigDialog(ui::ImGuiDrawer* imgui_drawer,
                       EmulatorWindow* emulator_window)
-      : ui::ImGuiDialog(imgui_drawer), emulator_window_(emulator_window) {
-    memset(gamertag, 0, sizeof(gamertag));
-  }
+      : ui::ImGuiDialog(imgui_drawer), emulator_window_(emulator_window) {}
+
+  static bool DrawProfileContent(ui::ImGuiDrawer* imgui_drawer,
```

Comment:
I figured we'd want "profile badge" thingy to be consistent across different UI areas so I turned this function into static one.